specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -12,7 +12,9 @@
     "application/json"
   ],
   "produces": [
-    "application/json"
+    "application/json",
+    "text/plain",
```

Comment:
Are you really returning these content types from the ARM control plane? text/plain and text/powershell. Please note that only json is supported.

Reply:
Yes, these are for the scripts and output as commented above. These are for read-only properties. These APIs are part of ARM since day 1 and signed off by ARM team like Charles, Anders, etc. I only fixed the swagger.